pkgs/tools/video/rav1e/default.nix

```diff
@@ -2,17 +2,17 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "rav1e";
-  version = "0.3.4";
+  version = "0.4.0-alpha";
 
   src = fetchFromGitHub {
     owner = "xiph";
     repo = "rav1e";
     rev = "v${version}";
-    sha256 = "0zwjg0sv504i1ahzfy2jgng6qwmyvcrvdrp4n3s90r4kvwjkv8xs";
+    sha256 = "1fw1gxi8330kfhl9hfzpn0lcmyn5604lc74d6g6iadzz2hmv4mb9";
   };
 
-  cargoSha256 = "1mfzshcbxky27nskxhcyrj99wd3v5f597ymgv7nb67lzp5lsyb24";
-
+  cargoPatches = [ ./cargo.patch ];
```

Comment:
That patch shouldn't be necessary (we try to avoid adding huge patches to Nixpkgs for various reasons).
You can use the old implementation from before e3fa38ec16314ab4fedd0d6305d81079c55d5cba ;)

Reply:
Done.
(You're welcome.)